braze/src/main/java/com/tealium/remotecommands/braze/BrazeInstance.java

```diff
@@ -154,10 +186,6 @@ public void initialize(@Nullable String apiKey, @Nullable JSONObject launchOptio
                 builder.setGreatNetworkDataFlushInterval(launchOptions.optInt(Config.GREAT_NETWORK_INTERVAL));
             }
 
-            if (BrazeUtils.keyHasValue(launchOptions, Config.IS_SDK_AUTHENTICATION_ENABLED)) {
-                builder.setIsSdkAuthenticationEnabled(launchOptions.optBoolean(Config.IS_SDK_AUTHENTICATION_ENABLED));
-            }
-
```

Comment:
how come this moved up 20 lines or so?

Reply:
Moved it up to be under the `// Booleans` comment. I had added in thinking it was new config option, but found it a few lines down so removed duplicate.